packages/shared/providers.yaml

```diff
@@ -1410,6 +1410,17 @@ highlevel-white-label:
     refresh_params:
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/highlevel
+holded:
+    categories:
+        - accounting
```

Comment:
Please add `holded` to the following individual categories in the documentation.

Reply:
Done!